.github/workflows/test.yml

```diff
@@ -67,20 +63,4 @@ jobs:
           os.system("gem install specific_install")
           os.system("gem specific_install -l https://github.com/{}.git -b {}".format(repo, branch))
 
-      - name: Metanorma compile windows
-        if: matrix.os == 'windows-latest'
-        run: |
-          choco install -y yq gnuwin32-coreutils.install
-          make -f Makefile.win clean all SKIP_BUNDLE=true SHELL=cmd
-
-      - name: Metanorma compile macos
-        if: matrix.os == 'macos-latest'
-        run: |
-          brew install yq
-          make clean all SKIP_BUNDLE=true
-
-      - name: Metanorma compile linux
-        if: matrix.os == 'ubuntu-latest'
-        run: |
-          sudo snap install yq
-          make clean all SKIP_BUNDLE=true
+      - run: metanorma site generate . -o published -c metanorma.yml --agree-to-terms
```

Comment:
And here.

Reply:
Yep, unfortunatelly it's broken because of https://github.com/metanorma/metanorma-cli/pull/224